client/my-sites/plugins/plugin-upload/index.jsx

```diff
@@ -80,12 +81,21 @@ class PluginUpload extends React.Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const {
+			translate,
+			upgradeJetpack,
+			siteId,
+		} = this.props;
 
 		return (
 			<Main>
 				<HeaderCake onClick={ this.back }>{ translate( 'Upload plugin' ) }</HeaderCake>
-				{ this.renderUploadCard() }
+				{ upgradeJetpack && <JetpackManageErrorPage
```

Comment:
We have both a `upgradeJetpack && ` and `! upgradeJetpack &&`, maybe that would make a good usage of the ternary operator?